packages/expect/src/jasmineUtils.ts

```diff
@@ -64,9 +64,9 @@ function asymmetricMatch(a: any, b: any) {
 function eq(
   a: any,
   b: any,
-  aStack: any,
-  bStack: any,
-  customTesters: any,
+  aStack: Array<unknown>,
```

Comment:
better typing makes me happy! 🙏

Reply:
Yes, he committed it better than he found it!

Reply:
> better typing makes me happy! 🙏

Same here 🙌